### PR TITLE
README in RestructedText for pypi

### PR DESCRIPTION
**Pull request by trojkat**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -32,13 +32,27 @@
...
 
-def read(fname):
-    """
-    Read filename.
 
-    :param str fname: name of a file to read
-    """
-    return open(os.path.join(here, fname)).read()
+try:
+    from pypandoc import convert
```
Comment (fizyk) [inline setup.py:36]: add to requirements

Comment (trojkat) [inline setup.py:36]: There's no need for that https://bitbucket.org/pypa/pypi/issues/148/support-markdown-for-readmes (or check ianitor)

**Comment by fizyk** [comment]
okay, pylama then :)

**Comment by tomislater** [comment]
Remember that in python 3, we can't do things like `from urllib import quote_plus, unquote_plus`. See documentation.

https://travis-ci.org/ClearcodeHQ/querystringsafe_base64/jobs/107802406

**Comment by fizyk** [comment]
@trojkat let's work on py3 in separate pull request, and turn it off for now.

**Comment by trojkat** [comment]
:+1:

File: .travis.yml [hunk 2]
```diff
@@ -2,11 +2,7 @@ language: python
 python:
   - 2.6
   - 2.7
-  - 3.3
```
Comment (tomislater) [inline .travis.yml:5]: You don't have to remove these guys. Look here: https://github.com/ClearcodeHQ/mirakuru/blob/master/.travis.yml#L32

You can run tests, but exclude them from results.

**Pull request merged**